classes/abstracts/ActionScheduler_Abstract_ListTable.php

```diff
@@ -561,7 +563,8 @@ public function column_default( $item, $column_name ) {
 	protected function display_header() {
 		echo '<h1 class="wp-heading-inline">' . esc_attr( $this->table_header ) . '</h1>';
 		if ( $this->get_request_search_query() ) {
-			echo '<span class="subtitle">' . esc_attr( $this->translate( sprintf( 'Search results for "%s"', $this->get_request_search_query() ) ) ) . '</span>';
```

Comment:
This code is very bad for i18n, since will send the string already with the placeholder filled into the `translate` method.